### DISCUSSION TITLE
SVG in template options

### DISCUSSION THREAD

**Discussion by iorbita** [templaza/astroid-framework#320] [General]
Hi,
it would be nice to be able to load SVG images in template options, especially for logos but also for background images.

Many thanks,
Lorenzo

**Comment by Chacapamac** [comment]
I @iorbita 
I just test in **Joomla 3 & 4** and I can see, and use svg in the **Astroid Template Manager**?

See in **Media Library** in Astroid **Template Manager**
![Screen Shot 2023-04-18 at 1 18 08 PM](https://user-images.githubusercontent.com/1907276/232856935-0a8bf18d-d07e-46fb-a21b-67c6801e1c28.jpg)

**Set in background** and work in pubic view (tested)
![Screen Shot 2023-04-18 at 1 30 17 PM](https://user-images.githubusercontent.com/1907276/232857617-f125221f-0e13-433f-80b9-61a1a37ad947.jpg)

**Comment by iorbita** [comment]
Hi @Chacapamac,
both in Astroid Template Manager and in public view (front-end), SVG image must have at least a defined width, _max-width_ property has no effect on SVG image, nor _%_ value.

http://u.pc.cd/giEitalK

Thanks,
Lorenzo

**Comment by Chacapamac** [comment]
@iorbita 
I think I know what you mean...
**You like to have the ability to enter a manual size in px,%, vh, vw, watever, Yeah, I think it can be useful in specific case?**

**Comment by iorbita** [comment]
Hi @Chacapamac  and thank you for your interest on this subject.
I think it would be interesting to add a defined width for thumbnail in Astroid Template Manager in order to be able to view SVG format. Also and why not, fields allowing to introduce image values that will displayed in front-end, this especially to avoid having images in SVG format with dimensions equal to 0 (which is the same reason why the thumbnail is not displayed in Astroid Template Manager…)

Many thanks,
Lorenzo

**Reply by Chacapamac** [reply]
Yes, I think it can be useful to have a bit more control Here
Let @sonvnn look at what he can do.
